### DISCUSSION TITLE
Executing channel from external app and github

### DISCUSSION THREAD

**Discussion by rangasamyv** [nextgenhealthcare/connect#4544] [Q&A] [answered]
Hi all, hope everyone is safe and doing good.

I'm completely new to Mirth and I'm using version 3.9.1. I need to address below requests, kindly help me on this.

1. I need to execute mirth channel(s) from my dotnet application. Do we have any mechanism to do it?
2. Can we maintain it in github to work more than one user at a time?
3. Any tutorial for deploying the channels into different environments? (deployment process)

Thanks.

**Comment by tonygermano** [comment, marked as answer]
You don't execute a channel. It runs like a service and accepts requests. I recommend reading the User Guide. There are several methods by which you could send a message to a mirth channel from a .NET application.
